docs/topics/Gateway.md

```diff
@@ -222,17 +222,31 @@ Sent by the client to indicate a presence or status update.
 
 | Field | Type | Description |
 |-------|------|-------------|
-| idle_since | ?integer | unix time (in milliseconds) of when the client went idle, or null if the client is not idle |
+| since | integer | unix time (in milliseconds) of when the client went idle, or null if the client is not idle |
 | game | ?object | either null, or an object with one key "name", representing the name of the game being played |
+| status | string | see [status types](#DOCS_GATEWAY/status-types), the new status of the client |
+| afk | boolean | whether or not the client is afk |
```

Comment:
Maybe this should be less ambiguous since there also is the `idle` status which you document as `client is afk` but this field has a different meaning as far as I know.